recipe/meta.yaml

```diff
@@ -29,10 +28,11 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - pybind11
-    - protobuf
+    - pybind11 2.10.4
+    # required >=3.20.2 but we have only 3.20.3 with python 3.11 support
```

Comment:
I'll be pedantic (and sorry in advance), but to be more precise, they specifically build with `3.20.2` and they allow `>=3.20.2` at runtime.

Reply:
We allow it at runtime too https://github.com/AnacondaRecipes/onnx-feedstock/pull/9/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR41. 
v3.20.3 is only for building a package